python/GafferImageUI/ImageWriterUI.py

```diff
@@ -250,6 +250,22 @@
 
 		],
 
+		"jpeg.chromaSubSampling" : [
+
+			"description",
+			"""
+			The chroma sub sampling used to write the jpg file.
+			Note that the file will be store as YCbCr instead of RGB.
```

Comment:
Just some grammar:

- "jpg" to "jpeg" ? Not sure, but the parent plug is called "jpeg" anyways"
- "stored" to "stored"